snap/snapcraft.yaml

```diff
@@ -124,25 +123,13 @@ apps:
       - network
       - network-bind
     stop-command: bin/kong-stop.sh
-  security-secrets-setup:
-    adapter: full
-    command: bin/security-secrets-setup -confdir $SNAP_DATA/config/security-secrets-setup/res generate
-    daemon: oneshot
-    environment:
-      SecretsSetup_DeployDir: $SNAP_DATA/secrets/
-      SecretsSetup_CertConfigDir: $SNAP_DATA/config/security-secrets-setup/res
-      XDG_RUNTIME_DIR: /tmp
-    start-timeout: 15m
   vault:
     adapter: none
-    after:
-      - consul
```

Comment:
Did your other PR to remove the vault consul backend land? If not, then I would leave consul here, and remove in the other PR.

Reply:
the other PR not land yet. I'll remove it after the other PR lands.

Reply:
this has been resovled